src/uikit.cs

```diff
@@ -8030,6 +8370,11 @@ interface UIImageSymbolConfiguration {
 
 		[Export ("isEqualToConfiguration:")]
 		bool IsEqualTo ([NullAllowed] UIImageSymbolConfiguration otherConfiguration);
+
+		[Watch (9,0), TV (16,0), MacCatalyst (16,0), iOS (16,0)]
+		[Static]
+		[Export ("configurationPreferringMonochrome")]
+		UIImageSymbolConfiguration GetConfigurationPreferringMonochrome ();
```

Comment:
Should this be a property instead of a method? It takes no parameters and returns a single value...

Reply:
I don't know, let me check with the rest of the API what is what we usually done. Is better to be consistent.